### PR TITLE
 add doc for experimental variable tidb_store_batch_size

### PR DESCRIPTION
**Pull request by you06**
Signed-off-by: you06 <you1474600@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

ref: pingcap/tidb#39361

Document the experimental feature `tidb_store_batch_size`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: pingcap/docs-cn#12252
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -3744,6 +3744,19 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
+> **Warning:**
+>
+> Currently,`tidb_store_batch_size` is an experimental feature. It is not recommended that you use it in production environments.
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
+- Type: Integer
+- Default value: `0`
+- Range: `[0, 25000]`
+- This variable is used to control the batch size of table reader Coprocessor Tasks of `IndexLookUp` executor. `0` means disable batch, you can increase it to optimize the extremely slow queries.
```
Comment (Oreoxmt) [inline system-variables.md:3758]:
```suggestion
- This variable is used to control the batch size of table reader Coprocessor Tasks of the `IndexLookUp` executor. `0` means disable batch. When the number of tasks is relatively large and slow queries occur, you can increase this variable to optimize the query.
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: system-variables.md [hunk 2]
```diff
@@ -3744,6 +3744,20 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
 - Unit: Seconds
 - This variable is used to set the refresh time of [statement summary tables](/statement-summary-tables.md).
 
+### `tidb_store_batch_size` <span class="version-mark">New in v6.5.0</span>
```
Comment (TomShawn) [inline system-variables.md:3747]:
```suggestion
### `tidb_store_batch_size`
```

File: system-variables.md [hunk 3]
```diff
@@ -3744,6 +3744,20 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
+
+> **Warning:**
+>
+> Currently,`tidb_store_batch_size` is not stable yet. This variable may be removed in the future, it is not recommended that you use it in production environments.
+> Suggest to use the default value.
```
Comment (TomShawn) [inline system-variables.md:3752]:
```suggestion
> Currently, `tidb_store_batch_size` is not stable yet. This variable might be removed in a future release. It is not recommended that you use it in production environments. It is not recommended to change the default value.
```

File: system-variables.md [hunk 4]
```diff
@@ -3744,6 +3744,20 @@ For details, see [Identify Slow Queries](/identify-slow-queries.md).
...
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
+- Type: Integer
+- Default value: `0`
+- Range: `[0, 25000]`
+- This variable is used to control the batch size of table reader Coprocessor Tasks of the `IndexLookUp` executor. `0` means disable batch. When the number of tasks is relatively large and slow queries occur, you can increase this variable to optimize the query.
```
Comment (TomShawn) [inline system-variables.md:3759]:
```suggestion
- This variable is used to control the batch size of the Coprocessor Tasks of the `IndexLookUp` operator. `0` means to disable batch. When the number of tasks is relatively large and slow queries occur, you can increase this variable to optimize the query.
```

**Review by TomShawn — APPROVED** [review]
minor comment

**Pull request merged**